distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -1106,30 +1177,40 @@ public void testDownloadFromCacheIfAvailable() throws Exception {
         when().get(restUrl).then().log().all().assertThat().contentType("text/plain")
                 .body(is(STUB_SERVER_TEST_STRING));
         // @formatter:on
-        cswServer.verifyHttp()
-                .times(1,
-                        Condition.uri("/services/csw"),
-                        Condition.parameter("request", "GetRecordById"));
+        cswServer.verifyHttp().times(1, Condition.uri("/services/csw"), Condition.parameter("request", "GetRecordById"));
+
+        expect("Waiting for notifications. Received " + cometDClient.getMessages(NOTIFICATIONS_CHANNEL).size() + " of 1").within(10, SECONDS).until(() -> cometDClient.getMessages(NOTIFICATIONS_CHANNEL).size() == 1);
```

Comment:
We need to get the Eclipse formatter fixed up for lambda expressions. This looks terrible.